docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -380,6 +382,8 @@ $firescrypt$ln=<mem_cost>,r=<rounds>,p=<parallelization>$<salt>$<hash>$<salt_sep
 $firescrypt$ln=14,r=8,p=1$sPtDhWcd1MfdAw==$xbSou7FOl6mChCyzpCPIQ7tku7nsQMTFtyOZSXXd7tjBa4NtimOx7v42Gv2SfzPQu1oxM2/k4SsbOu73wlKe1A==$Bw==$YE0dO4bwD4JnJafh6lZZfkp1MtKzuKAXQcDCJNJNyeCHairWHKENOkbh3dzwaCdizzOspwr/FITUVlnOAwPKyw==
 ```
 
+Documentation for extracting your users can be found [here](https://firebase.google.com/docs/cli/auth)
```

Comment:
Let me know if you think this should live under a separate block instead of the "Example" block. Wanted to follow the structure of the other tabs.